src/runtime/internal/constants.ts

```diff
@@ -0,0 +1 @@
+export const resolved_promise = Promise.resolve();
```

Comment:
[ask]

Why `Promise.resolve();` can be constsnt?

`Promise.resolve() === Promise.resolve()` returns false, so I didn't get why using same promise is ok.

Reply:
`Promise.resolve()` is a way to schedule something that should run before the next JavaScript event loop iteration. You can chain as many `.then`s (and `await`s) as you want on a promise. Therefore we can use the same object over and over again.

Reply:
Are callbacks correctly freed after execution?
If not this may lead in huge memory leak...

Reply:
> a way to schedule something that should run before the next JavaScript event loop iteration

[`queueMicrotask`](https://developer.mozilla.org/en-US/docs/Web/API/queueMicrotask)?